projects/addon-doc/components/navigation/navigation.template.html

```diff
@@ -19,6 +19,7 @@
                     *ngFor="let item of group"
                     rel="noopener noreferrer"
                     tuiOption
+                    tuiRouterLink
```

Comment:
I'm not sure I like this directive. Maybe we can just check if path contains :// and use regular a href instead?

Reply:
fixed